hackathon_site/review/jinja2/review/send-decisions.html

```diff
@@ -0,0 +1,53 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}SEND DECISION EMAILS TO PARTICIPANTS{% endblock %}
+{% block form_attrs %}enctype="multipart/form-data"{% endblock %}
+
+{% set form_action=url("admin:send-decision-emails") %}
+
+{% block form_description %}
+    {% if form.non_field_errors %}
+        <div class="row col s12">
+        {% for error in form.non_field_errors() %}
+            <span class="formError">{{ error }}</span>
+        {% endfor %}
+        </div>
+        <div class="row col s12">
+            Accepted: {{ accepted_count }}
+            Waitlisted: {{ waitlisted_count }}
+            Rejected: {{ rejected_count }}
+        </div>
+    {% endif %}
+{% endblock %}
+
+{% block form_inputs %}
+    {% for field in form %}
+        {% if field.field.widget.input_type == "select" %}
```

Comment:
This doesn't seem to be necessary. `render_field(field, size="s12")` works fine for the select field on my end